clients/python-client/examples/use-raw-with-api.py

```diff
@@ -0,0 +1,204 @@
+                    ],
+                    "volumes": [{"name": "ray-logs", "emptyDir": {}}],
+                },
+            },
+        },
+        "workerGroupSpecs": [
+            {
+                "replicas": 1,
+                "minReplicas": 1,
+                "maxReplicas": 10,
+                "groupName": "small-group",
+                "rayStartParams": {"block": "true"},
+                "template": {
+                    "spec": {
+                        "containers": [
+                            {
+                                "name": "ray-worker",
+                                "image": "rayproject/ray:2.2.0",
+                                "lifecycle": {
+                                    "preStop": {
+                                        "exec": {
+                                            "command": ["/bin/sh", "-c", "ray stop"]
+                                        }
+                                    }
+                                },
+                                "volumeMounts": [
+                                    {"mountPath": "/tmp/ray", "name": "ray-logs"}
+                                ],
+                                "resources": {
+                                    "limits": {"cpu": "1", "memory": "1G"},
+                                    "requests": {"cpu": "500m", "memory": "1G"},
+                                },
+                            }
+                        ],
+                        "initContainers": [
+                            {
+                                "name": "init",
+                                "image": "busybox:1.28",
+                                "command": [
+                                    "sh",
+                                    "-c",
+                                    "until nslookup $RAY_IP.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for K8s Service $RAY_IP; sleep 2; done",
+                                ],
+                            }
+                        ],
+                        "volumes": [{"name": "ray-logs", "emptyDir": {}}],
+                    }
+                },
+            }
+        ],
+    },
+}
+
+
+def main():
+
+    print("starting cluster handler...")
+
+    my_kube_ray_api = kuberay_cluster_api.RayClusterApi()
+
+    my_kube_ray_api.create_ray_cluster(body=cluster_body)
+
+    my_kube_ray_api.create_ray_cluster(body=cluster_body2)
+
+    # the rest of the code is simply to fetch, print and cleanup the created cluster
+    kube_ray_cluster = my_kube_ray_api.get_ray_cluster(
+        name=cluster_body["metadata"]["name"], k8s_namespace="default"
+    )
+
+    if kube_ray_cluster:
+        print("printing the raycluster json representation...")
+        json_formatted_str = json.dumps(kube_ray_cluster, indent=2)
+        print(json_formatted_str)
+
+    print(
+        "try: kubectl -n default get raycluster {} -oyaml".format(
```

Comment:
Typo
```suggestion
        "try: kubectl -n default get raycluster {} -o yaml".format(
```

Reply:
`-oyaml` and `-o yaml` are equivalent, I fixed it to avoid any confusion :-)
fixed